### DISCUSSION TITLE
AR Effects DeepAR - Is it possible?

### DISCUSSION THREAD

**Discussion by AlexUrrutia** [pedroSG94/RootEncoder#1190] [Ideas]
Hello @pedroSG94 , can we use this https://www.deepar.ai/ with your library to add AR effects?

Thanks!
![Screen Shot 2022-11-10 at 10 10 52 AM](https://user-images.githubusercontent.com/7042361/201132243-25378cb7-2dba-4f84-8fea-16d46650674a.png)

**Comment by pedroSG94** [comment]
Hello,

I don't know how that library is working so I can't response you about it.
Maybe you can do it modifying the library or using only rtmp/rtsp module if you have a way to get aac and h264 data
